spec/keycard/request/cosign_attributes_spec.rb

```diff
@@ -41,18 +41,17 @@
       expect(attributes.user_eid).to be_nil
     end
 
-    it "the client_ip is empty (though this would be a proxy config error)" do
+    it "the client_ip is empty" do
```

Comment:
Where could this advisory notion (config error) go instead of here? This parenthetical was a little piece of unhappy path documentation that I'd rather not lose.

Reply:
I actually just copied the ProxyAttributes specs into this file. I can add this to both.